src/plugins/kss.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2017-present Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain
+ * a copy of the license in the LICENSE and PATENTS files in the root of this
+ * repository.
+ */
+
+import * as kss from "kss";
+import * as path from "path";
+import { ICompiler, IFile, IPlugin } from "./plugin";
+
+/** A single modifier for an example. */
+export interface IKssModifier {
+    documentation: string;
+    name: string;
+}
+
+/**
+ * A markup/modifiers example parsed from a KSS comment block.
+ */
+export interface IKssExample {
```

Comment:
given that the method that returns this is `convertSection` and using the name "example" might be confusing with the `@example` tag, i vote this should be named `IKssSection`